src/psyclone/psyir/transformations/parallel_loop_trans.py

```diff
@@ -310,15 +331,39 @@ def apply(self, node, options=None):
             validate and collapse steps have failed.
 
         '''
+        # TODO 2668 - options dict is deprecated.
         if not options:
-            options = {}
-        self.validate(node, options=options)
-
-        verbose = options.get("verbose", False)
-        collapse = options.get("collapse", False)
-        ignore_dep_analysis = options.get("force", False)
-        list_of_names = options.get("ignore_dependencies_for", [])
-        privatise_arrays = options.get("privatise_arrays", False)
+            self.validate_options(
+                    verbose=verbose, collapse=collapse,
+                    ignore_dependencies_for=ignore_dependencies_for,
+                    privatise_arrays=privatise_arrays,
+                    sequential=sequential, **kwargs
+            )
+            # Rename the input options that are renamed in this apply method.
+            # TODO 2668, rename options to be consistent.
+            ignore_dep_analysis = force
+            if ignore_dependencies_for is None:
+                list_of_names = []
+            else:
+                list_of_names = ignore_dependencies_for
+        else:
+            if len(options.keys()) > 0:
+                print("Deprecation Warning: The options parameter to "
```

Comment:
Use message from a class variable perhaps?

Reply:
Using the one inherited from Transformation.